src/index.html

```diff
@@ -7,6 +7,278 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
```

Comment:
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/38065883/173182975-f5355121-ffa0-4122-81d7-0a8984e11805.png">
Did you deploy your work with `npm run deploy` after you've made changes to your code?

Reply:
Yes, and I did it one more time